audio_service/lib/audio_service.dart

```diff
@@ -179,9 +193,9 @@ class PlaybackState {
         this.updateTime = updateTime ?? DateTime.now();
 
   /// Creates a copy of this state with given fields replaced by new values,
-  /// with [updateTime] set to [DateTime.now()], and unless otherwise replaced,
-  /// with [updatePosition] set to [this.position]. [errorCode] and
-  /// [errorMessage] will be set to null unless [processingState] is
+  /// and unless otherwise replaced, with [updatePosition] set to [position].
```

Comment:
Why remove the documentation of the updateTime behaviour?

Reply:
i didn't remove it, i corrected it
previous doc was wrong if you look at the code

Reply:
Not following. The documentation was correct, so if the code is different, then the code is wrong (although I doubt that it is wrong, the update time should be initialised to `DateTime.now()` as if it weren't, none of the position events would sync up correctly.)

Reply:
updateTime is initialized in the constuctor, not the `copyWith` method. previous doc was contradicting itself by saying that it's both `now()` and replaced with position

Reply:
`copyWith` *uses* the constructor, so yes it has this behaviour as a black box. It is critical to document this so that the user understand that this will be set as such.

Reply:
fixed